microsoft/testsuites/core/storage.py

```diff
@@ -294,21 +300,34 @@ def verify_os_partition_identifier(self, log: Logger, node: RemoteNode) -> None:
         )
         os_partition_info = node.tools[Blkid].get_partition_info_by_name(os_partition)
 
+        # check if cvm
+        is_cvm = False
+        settings = Feature.get_feature_settings(
+            node.features[SecurityProfile].get_settings()
+        )
+        assert isinstance(settings, SecurityProfileSettings)
+        if SecurityProfileType.CVM == settings.security_profile:
+            log.debug("This is CVM.")
```

Comment:
The description should be more technical like "Confidential computing is detected to be enabled."